web/src/views/SubmissionPortal/Components/LoginPrompt.vue

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+
+export default defineComponent({
+  props: {},
+  setup() {
+    return {
+      loginHref: '/login',
+    };
+  },
+});
+</script>
+
+<template>
+  <v-alert
+    class="mb-0"
+    text="text"
+  >
+    <v-container>
+      <v-row align="center">
+        <v-col class="justify-content-center">
+          <p class="body-1">
+            Login with your ORCiD
+            to access the Submission Portal.
+          </p>
+          <p class="body-2 mb-0 text--disabled">
+            Don't have an ORCiD? Individuals can
+            <a
+              href="https://support.orcid.org/hc/en-us/articles/360006973953"
+              target="_blank"
+              title="Learn about registering an ORCiD"
+              class="text--disabled"
+            >get one for free</a>.
+          </p>
```

Comment:
I am concerned about the readability of this text. The contrast from the background is low.

Reply:
Thank you for bringing this to my attention, @marySalvi!

I plugged in the colors into [an online tool](https://webaim.org/resources/contrastchecker/) and got "failing grades":

![image](https://github.com/microbiomedata/nmdc-server/assets/134325062/2b7a72c1-a73f-45b2-b5ba-92afab62b8a2)

I will increase the contrast (TBD).

Reply:
I changed it to black in commit 6865d7c86fe6bb33a5c764a6645b0537b32be8a0. That online tool shows "passing" all across the board, for black (on gray).